src/select/stateful-select-container.js

```diff
@@ -41,6 +42,15 @@ class StatefulSelectContainer extends React.Component<
     return onChange(e, params);
   };
 
+  onTextInputChange = (e: SyntheticInputEvent<HTMLInputElement>) => {
+    this.stateReducer(STATE_CHANGE_TYPE.textChange, e, {
```

Comment:
Would it make sense to use `internalSetState` name here to align with [other components](https://github.com/uber-web/baseui/blob/master/src/popover/stateful-container.js#L85)?